app/_locales/en/messages.json

```diff
@@ -283,6 +283,9 @@
   "addNewBitcoinAccount": {
     "message": "Add a new Bitcoin account (Beta)"
   },
+  "addNewBitcoinTestnetAccount": {
```

Comment:
Do we have translations for these messages on `develop` yet? Not ideal to ship this with those missing

Reply:
Not yet, but this feature is only available on Flask, not sure if that is relevant for the translation case